content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -91,7 +93,8 @@ To list all the Pods that belong to a Job in a machine readable form, you can us
 pods=$(kubectl get pods --selector=job-name=pi --output=jsonpath='{.items[*].metadata.name}')
 echo $pods
 ```
-```
+The output is similar to this:
+```shell
```

Comment:
This line is incorrect
It should be: <tt>\`\`\`</tt>

Reply:
Thanks for your guidance.